<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatriciaTree</class>
 <widget class="QMainWindow" name="PatriciaTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatriciaTree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelInsert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Insertar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelete">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInsert">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonInsert">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelete">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Patricia Tree Implementation</string>
    </property>
   </widget>
   <widget class="QLabel" name="members">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Integrantes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="members_2">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Javier Olivares</string>
    </property>
   </widget>
   <widget class="QLabel" name="members_3">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matias Meneses</string>
    </property>
   </widget>
   <widget class="QLabel" name="members_4">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Santiago Silva</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInsert_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encontrar todos los valores:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonFindAll">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Encontrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelete_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encontrar raíz:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonFindRoot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Encontrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar árbol:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonVaciar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alfabeto a utilizar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlf1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlf2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAlf">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Usar</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>1491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
